apps/meteor/tests/end-to-end/api/05-chat.js

```diff
@@ -1346,39 +1346,6 @@ describe('[Chat]', function () {
 		});
 	});
 
-	describe('[/chat.getMessageReadReceipts]', () => {
```

Comment:
Why were those testes removed?

Reply:
It's referenced here to add a new file to cover both EE and CE, I believe:
https://github.com/RocketChat/Rocket.Chat/pull/27074/commits/a31375aeeeb33afe1c05aa22cd1f4c757848722d

Since this PR is supposed to go to the 6.0, I don't think it's viable to do these tests today

Reply:
Yeah, not only I think this should be in a separate file, but also this needs to be updated since this endpoint is now EE-only. I plan to get the tests I worked on #26611 (old PR related to this issue), but these also need to be changed since this feature has been moved to EE
I'm planning to add this as a regression later so as not to hold this PR

Reply:
Thanks @LucianoPierdona for adding this :hugs: